opds.py

```diff
@@ -109,11 +86,11 @@ def group_uri(cls, work, license_pool, identifier):
     @classmethod
     def rating_tag(cls, type_uri, value):
         """Generate a schema:Rating tag for the given type and value."""
-        rating_tag = E._makeelement("{%s}Rating" % schema_ns)
-        value_key = '{%s}ratingValue' % schema_ns
+        rating_tag = AtomFeed.E._makeelement("{%s}Rating" % AtomFeed.SCHEMA_NS)
```

Comment:
This change makes the code uglier than before, but it would become less ugly than before if you defined `AtomFeed.makeelement()` as a shortcut for this.